repo/format_block.go

```diff
@@ -17,6 +17,7 @@ import (
 )
 
 const (
+	aes256GcmEncryption             = "AES256_GCM"
 	defaultFormatEncryption         = "AES256_GCM"
```

Comment:
nit: use defaultFormatEncryption = aes256GcmEncryption 

also, Gcm should be spelled GCM per Golang convention.

Reply:
I tried this earlier, but the linter was getting confused, not yet sure why. Might have to file a linter bug.